src/main/g8/src/test/resources/$package$/$name__camel$/infosets/test_01.$name__camel$.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
You might want to remove the unnecessary whitespace character before the `?>`.

Reply:
Good eye, will fix.